.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,21 @@
+---
+name: "\U00002753Questions/Help/Support"
+about: Please let us know if you have any questions.
+title: ''
+labels: ''
```

Comment:
If it's worth creating a separate template (for help and support), should we also create a corresponding label?

Reply:
I think that it's nice to create a new label for this template.
Do you have any thoughts about the new label name? 
(`question` which is one of the default GitHub labels may be used for this purpose.)

Reply:
Great. `question` sounds good to me as it somewhat reflects the intent of this template.

Reply:
Thanks. I added `qustion` label to this template (https://github.com/pfnet/optuna/pull/652/commits/8299aa14f02df23c50af157d699394cab5e2ecdf).